Translation corrections (#104)
07th-mod/higurashi-console-arcs: Update/some_008.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300416", 256, TRUE);
	OutputLine(NULL, "「魚か私は？！",
			NULL, "\"Are you saying I'm like a fish?!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300417", 256, TRUE);
	OutputLine(NULL, "　冗談に決まってるでしょうがっ」",

> 			NULL, " I thought it was obvious I was kidding.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I was kidding, obviously!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400178", 256, TRUE);
	OutputLine(NULL, "「冗談だとしても、時と場合を選びなさいっての。",

> 			NULL, "\"Even if you're joking, you need to consider the time and place.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"There's a time and place for every joke.", Line_WaitForInput);
```

	ModDrawCharacter(4, 38, "sprite/tama1_kanashimi_", "1", -160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 19, 200, TRUE );

	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400179", 256, TRUE);
	OutputLine(NULL, "…それにしても、大丈夫？",

> 			NULL, " ...Anyway, are you all right?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " ...Anyway, Natsumi, are you all right?", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400180", 256, TRUE);
	OutputLine(NULL, "　夏美の顔色、あんまり良くないよ？」",

> 			NULL, " Natsumi, your complexion doesn't look very healthy, you know?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Your face looks pale, you know?\"", Line_Normal);
```

	ClearMessage();

//r心配げに『ちょっとごめん』と断ってから、珠子ちゃんはそっと手を私の額に当てた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "心配げに『ちょっとごめん』と断ってから、珠子ちゃんはそっと手を私の額に当てた。",

> 			NULL, "Looking concerned, Tamako-chan quickly apologized then softly held her hand to my forehead.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Looking concerned, Tamako-chan motioned me to hold still for a moment, then gently placed her hand on my forehead.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300418", 256, TRUE);
	OutputLine(NULL, "「そうだね、すぐ電話してみるよ。",

> 			NULL, "\"Of course, I'll call right away.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Good idea, I'll give it a call right away.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100673", 256, TRUE);
	OutputLine(NULL, "「い、いいよぉ～。",

> 			NULL, "\"I-I'm fine, ok~.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I-It's ok~.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100674", 256, TRUE);
	OutputLine(NULL, "私、全然平気だから…」",

> 			NULL, " I'm completely all right, so...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I'm totally fine, so...\"", Line_Normal);
```

	ClearMessage();

	FadeAllBustshots( 500, TRUE );

//r私は慌てて遠慮したが、千紗登ちゃんは遠慮はなし、とばかりに手を振って、教室を出て行く。それを追いかけようとして席を立つと、珠子ちゃんは肩を押さえてぽん、と私を強引に席へと戻した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私は慌てて遠慮したが、千紗登ちゃんは遠慮はなし、とばかりに手を振って、教室を出て行く。それを追いかけようとして席を立つと、珠子ちゃんは肩を押さえてぽん、と私を強引に席へと戻した。",

> 			NULL, "I tried to stop her but Chisato-chan wouldn't hold back. She waved at me and ran out of the classroom. When I got up to follow her, Tamako-chan grabbed my shoulders and pushed me back into my seat.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I tried to stop her, but Chisato shook her head with a friendly smile, then waved at me and ran out of the classroom. When I got up to follow her, Tamako-chan grabbed my shoulders and pushed me back into my seat.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400183", 256, TRUE);
	OutputLine(NULL, "「いいから座ってなさい、夏美。",

> 			NULL, "\"It's ok Natsumi, please stay seated.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Just stay where you are, Natsumi.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100675", 256, TRUE);
	OutputLine(NULL, "「大げさだってば…」",

> 			NULL, "\"You're exaggerating...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"You're blowing this out of proportion...\"", Line_Normal);
```

	ClearMessage();

//rそういって苦笑を返しても、珠子ちゃんは『だめ』と静かに首を横に振る。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そういって苦笑を返しても、珠子ちゃんは『だめ』と静かに首を横に振る。",

> 			NULL, "Even though I said that with a forced smile, Tamako-chan replied \"no good\" while shaking her head at me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I said that with a forced smile, but Tamako-chan just shook her head at me.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400186", 256, TRUE);
	OutputLine(NULL, "「ウチの弟たちもね、風邪引きかけでも平気だ何だって無理して、余計にこじらせるんだよね」",

> 			NULL, "\"I've seen it with my younger brothers. Failing to take the cold weather into consideration can make things worse for yourself, you know.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"I've seen it with my younger brothers. They insist they're fine even when they catch colds, and it just makes things worse for them, you know.\"", Line_Normal);
```

	ClearMessage();

//夏美rvS04/36/310100676.「珠子ちゃん、私、同い年なんだけど…」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100676", 256, TRUE);
	OutputLine(NULL, "「珠子ちゃん、私、同い年なんだけど…」",

> 			NULL, "\"Tamako-chan, we're supposed to be the same age, but...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Tamako-chan, we're supposed to be the same age...\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400187", 256, TRUE);
	OutputLine(NULL, "「世話のかかり度合いで言えば同じレベルよ。",

> 			NULL, "\"When it comes to matters of personal health, we are talking on the same level.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Everyone gets treated the same way in times of ill health.", Line_WaitForInput);
```

//r無言で肩をすくめる。…口惜しいけど、仰る通りです。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "無言で肩をすくめる。…口惜しいけど、仰る通りです。",

> 			NULL, "I silently shrugged my shoulders. ...It was annoying, but what she said was true.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I silently shrugged my shoulders. ...I hated to admit it, but she was right.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100685", 256, TRUE);
	OutputLine(NULL, "「ごめん、わがまま言って。",
			NULL, "\"Sorry, I know I'm being selfish.", Line_WaitForInput);
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100686", 256, TRUE);
	OutputLine(NULL, "…すぐ終わるから、ね？」",
			NULL, " ...I'll be quick, ok?\"", Line_Normal);
	ClearMessage();

//r申し訳ない気分で謝りながら、私は手を合わせる。千紗登ちゃんはやれやれと肩をすくめると、気乗りのしない声のまま『このままで』と運転手の山科さんに言ってくれた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "申し訳ない気分で謝りながら、私は手を合わせる。千紗登ちゃんはやれやれと肩をすくめると、気乗りのしない声のまま『このままで』と運転手の山科さんに言ってくれた。",
			NULL, "I clapped my hands together and apologized. Chisato-chan shrugged her shoulders in exasperation, then with a dull voice told the driver Yamashina-san \"keep going.\"", Line_Normal);
	ClearMessage();

//r…私が家に帰る前に、介護福祉センターに立ち寄ってほしいとお願いしたとき、千紗登ちゃんはあまりいい顔をしなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…私が家に帰る前に、介護福祉センターに立ち寄ってほしいとお願いしたとき、千紗登ちゃんはあまりいい顔をしなかった。",

> 			NULL, "...Chisato-chan didn't look the least bit thrilled by my request to stop by the care and welfare center before I headed home.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Chisato-chan didn't look the least bit thrilled with my request to stop by the care and welfare center before I headed home.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300422", 256, TRUE);
	OutputLine(NULL, "『今日でなくてもいいんじゃない？",
			NULL, "\"You don't have to go today, you know?", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300423", 256, TRUE);
	OutputLine(NULL, "　今週は水曜も午前授業で働くんだから、その時でも遅くないでしょ？』",
			NULL, " You'll be working during morning classes on Wednesday too, so you can just wait until then, right?\"", Line_Normal);
	ClearMessage();

//r千紗登ちゃんはそう言って私を諭したが、…やっぱり今日行っておきたかったので、無理を聞いてもらうことにした。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "千紗登ちゃんはそう言って私を諭したが、…やっぱり今日行っておきたかったので、無理を聞いてもらうことにした。",

> 			NULL, "Chisato-chan tried talking me down... But I wanted to go today, so I made an unreasonable request.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Chisato-chan tried talking me down... but I wanted to go today, so I made an unreasonable request.", Line_Normal);
```

//rこぶしを振り上げて力説する千紗登ちゃんの姿に、相変わらずと思いながらもやんわりと突っ込む。そこへ予想外にも、運転席の山科さんから声が飛んできた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "こぶしを振り上げて力説する千紗登ちゃんの姿に、相変わらずと思いながらもやんわりと突っ込む。そこへ予想外にも、運転席の山科さんから声が飛んできた。",

> 			NULL, "I try to gently let Chisato down as she emphatically thrust her fist into the air and pontificates. Just then, Yamashima-san chimes up for once in the driver's seat.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I tried to gently let Chisato down as she emphatically thrust her fist into the air and pontificated. Just then, Yamashima-san suddenly chimed in from the driver's seat.", Line_Normal);
```

	ClearMessage();

//山科rvS04/00/VT1B_yamash1002.「…お嬢様にならともかく、旦那様に恨みを持つ方はちょっと考えられませんね」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>山科</color>", NULL, "<color=#f5e6d3>Yamashina</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_yamash1002", 256, TRUE);
	OutputLine(NULL, "「…お嬢様にならともかく、旦那様に恨みを持つ方はちょっと考えられませんね」",

> 			NULL, "\"...While you may be a different story, young mistress, I can't imagine anyone holding a grudge against the master.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...For a lady like you, I can't imagine what would happen to someone with a grudge against your husband.\"", Line_Normal);
```
